common/common-lang/src/main/java/com/twelvemonkeys/lang/BeanUtil.java

```diff
@@ -106,7 +106,7 @@ public static Object getPropertyValue(Object pObject, String pProperty) {
             Class[] paramClass = new Class[0];
 
             int begBracket;
-            if ((begBracket = subProp.indexOf("[")) > 0) {
+            if ((begBracket = subProp.indexOf('[')) > 0) {
```

Comment:
indexOf on char should be ever so slightly faster than on String. Less instructions, smaller object. Really small stuff though. Still, why not.